src/Compilers/Core/Portable/CommandLine/AnalyzerConfigSet.cs

```diff
@@ -9,6 +9,7 @@
 using System.Diagnostics;
 using System.Diagnostics.CodeAnalysis;
 using System.IO;
+using System.Linq;
```

Comment:
Why was this change needed?

Reply:
Because net8.0 has `IEnumerable<T>.Order` extension method, so I disabled the one from Roslyn utilities in net8.0, but here the `EnumerableExtensions` weren't visible.

<samp>roslyn\src\Compilers\Core\Portable\CommandLine\AnalyzerConfigSet.cs(553,58): error CS1061: 'IEnumerable<string>' does not contain a definition for 'Order' and no accessible extension method 'Order' accepting a first argument of type 'IEnumerable<string>' could be found (are you missing a using directive or an assembly reference?) [roslyn\src\Compilers\Core\Portable\Microsoft.CodeAnalysis.csproj::TargetFramework=net8.0]</samp>